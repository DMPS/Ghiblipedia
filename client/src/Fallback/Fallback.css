.fallback {
  background-image: url(./error-image.jpg);
  width: 100%;
  height: 100vh !important;
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
}

.header {
  color: white;
  font-size: 1.5em;
}
