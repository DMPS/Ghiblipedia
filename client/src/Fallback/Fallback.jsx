import React from "react";
import "./Fallback.css";

export default () => {
  return (
    <div className="fallback">
      <h1 className="header">
        I'm sorry. I'm busy learning how to multiply. Please come back later.
      </h1>
    </div>
  );
};
