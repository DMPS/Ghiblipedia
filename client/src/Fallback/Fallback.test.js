import React from "react";
import Fallback from "./Fallback";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const tree = renderer.create(<Fallback />).toJSON();
  expect(tree).toMatchSnapshot();
});
