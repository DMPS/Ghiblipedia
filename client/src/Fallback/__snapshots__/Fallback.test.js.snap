// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="fallback"
>
  <h1
    className="header"
  >
    I'm sorry. I'm busy learning how to multiply. Please come back later.
  </h1>
</div>
`;
